src/components/FindPark.vue

```diff
@@ -30,23 +30,12 @@ export default {
   data() {
     return {
       searchTerm: '',
-      inputIsInvalid: false,
     };
   },
   methods: {
-    setSearchTerm() {
```

Comment:
Glad we could remove this method, due to the fact that our button is only rendered if our input is valid.